src/debuggee.ts

```diff
@@ -34,10 +34,10 @@ export interface DebuggeeProperties {
 }
 
 export class Debuggee {
-  private uniquifier: string;
-  private description: string;
-  private agentVersion?: string;
-  private sourceContexts?: Array<{[key: string]: any}>;
+  uniquifier: string;
```

Comment:
👍

Reply:
Why are these no longer private?

Reply:
Current compiler will complain their values were never read. However, we cannot remove them as they need to be send to debug server in wire. Since it needs to be exposed to debug server anyway, I made them public.

Reply:
`Debuggee` might be better off as an interface, but that's out of the scope of this PR.